<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstConfig</class>
 <widget class="QDialog" name="InstConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>1307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="PushButton" name="TestB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="CardWidget" name="CurrCardWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>330</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="StrongBodyLabel" name="CurrSettingLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>电流通道配置</string>
    </property>
   </widget>
   <widget class="CaptionLabel" name="CaptionLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>通道列表</string>
    </property>
   </widget>
   <widget class="ComboBox" name="CurrType">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>AC/DC</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ComboBox" name="CurrRange">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>电流范围</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="PushButton" name="CurrConfButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>提交配置</string>
    </property>
   </widget>
   <widget class="PlainTextEdit" name="CurrChannelList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>160</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="CardWidget" name="TempCardWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>330</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="StrongBodyLabel" name="TempSettingLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温度通道配置</string>
    </property>
   </widget>
   <widget class="CaptionLabel" name="CaptionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>通道列表</string>
    </property>
   </widget>
   <widget class="ComboBox" name="ProbeType">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>探头类型</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ComboBox" name="TemperatureUnit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>温度单位</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ComboBox" name="SensorType">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>传感器型号</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="PushButton" name="TempConfButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>提交配置</string>
    </property>
   </widget>
   <widget class="PlainTextEdit" name="TempChannelList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>160</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="CaptionLabel" name="CaptionLabel_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>配置选项</string>
    </property>
   </widget>
  </widget>
  <widget class="CardWidget" name="InstSettingCardWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>330</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="PushButton" name="InitInstButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="LineEdit" name="ScanInterval">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>120</width>
      <height>33</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>扫描间隔(s)</string>
    </property>
    <property name="whatsThis">
     <string>扫描间隔(s)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="StrongBodyLabel" name="InstSettingLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>21</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>仪器配置</string>
    </property>
   </widget>
   <widget class="CheckBox" name="reset">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>重置仪器</string>
    </property>
   </widget>
   <widget class="CheckBox" name="Cls">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>清除显示</string>
    </property>
   </widget>
   <widget class="ComboBox" name="Timeout">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>129</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>超时时间(s)</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ComboBox" name="SetCom">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>89</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="ComboBox" name="Baudrate">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>169</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="PushButton" name="CheckInstButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>自检</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
